src/components/WrapCryptoList/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React, {ReactNode} from 'react';
+import styled from 'styled-components/native';
+
+interface WrapCryptoList {
+  children: ReactNode;
+}
+
+const WrapCryptoList = ({children}: WrapCryptoList): JSX.Element => {
+  return <WrapList>{children}</WrapList>;
+};
+export default WrapCryptoList;
+
+const WrapList = styled.ScrollView`
```

Comment:
styles should be stored in a different file

Reply:
Okey.